shuup/order_printouts/templates/shuup/order_printouts/admin/macros.jinja

```diff
@@ -26,10 +26,12 @@
             <td>{{ order.get_shipping_status_display() }}</td>
         </tr>
         {% if tracking_codes %}
+            {% for tracking_code in tracking_codes %}
             <tr>
-                <td>{% trans %}Tracking codes{% endtrans %}</td>
-                <td>{{ tracking_codes|join(', ') }}</td>
+                <td>{% trans %}Tracking code{% endtrans %} #{{ loop.index }}:</td>
```

Comment:
make sure to update translation strings too `shuup_makemessages -l en` and use the index in the translations message:
```jinja
{% set index = loop.index %}
{% trans index=index %}Tracking code #{{ index }}:{% endtrans %}
```

Reply:
Done